### PR TITLE
Remove Ember.K.

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: app/components/drag-handle.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import Ember from "ember";
-const { computed, String: { htmlSafe }, K } = Ember;
+
+const { computed, String: { htmlSafe } } = Ember;
+const K = function() { return this; }
```
Comment (rwjblue) [inline app/components/drag-handle.js]: IMHO, the replacement here is simply to use empty functions where needed. We don't need another "special" Emberism, let's just use functions.

**Comment by stefanpenner** [comment]
👍

File: app/mixins/row-events.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ export default Mixin.create({
    * @type {Function}
    * @default Ember.K
```
Comment (sduquej) [inline app/mixins/row-events.js]: 🔪

File: app/mixins/row-events.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ export default Mixin.create({
...
```
Comment (sduquej) [inline app/mixins/row-events.js]: 🔪

File: app/mixins/row-events.js [hunk 4]
```diff
@@ -27,7 +27,7 @@ export default Mixin.create({
...
```
Comment (sduquej) [inline app/mixins/row-events.js]: 🔪

File: app/components/drag-handle.js [hunk 5]
```diff
@@ -45,7 +47,7 @@ export default Ember.Component.extend({
    * @type {Function}
    * @default Ember.K
```
Comment (sduquej) [inline app/components/drag-handle.js]: 🔪

**Comment by locks** [comment]
Done!

**Pull request merged**